src/Async/index.js

```diff
@@ -322,6 +322,37 @@ function Async(fn) {
     }
   }
 
+  function bichain(method) {
```

Comment:
This will not be covered by `fantasy-land` no need to set up the methodname

Reply:
kk awesome, i actually was unsure what it was for but thought best to copy the convention. I thought it might have been for future refactor or something